### ISSUE TITLE
EASY FEATURE REQUEST: Player-names in players-tracker

### ISSUE THREAD

**Issue by i12know** [jonatkins/ingress-intel-total-conversion#1077]
Would it be possible to have a new plug-in: Player-names, which would show the player's names on the player-tracker's feature? I am using a 2nd computer for Intel screen and would not want to mouseover to find out who is moving across the screen.  Thanks.

**Comment by flamusdiu** [comment]
Yes and no. i think that would cause a lot extra text on the intel causing issue with other items.

**Comment by neon-ninja** [comment]
I had a crack at this, with my strategy being forcing all tooltips to display at once. Like @flamusdiu says, it makes it super messy, but if you're still keen to use it you can install it from here https://gist.github.com/neon-ninja/c51dd249b214a1f88027

Screenie:

![player tracker names](https://cloud.githubusercontent.com/assets/3378822/12226476/8468e6b0-b881-11e5-95ac-78903a579c9c.png)

**Comment by neon-ninja** [comment]
I had another attempt, this time using the leaflet label plugin

https://gist.github.com/neon-ninja/0ef22dae138d31604983

![player tracker names alt](https://cloud.githubusercontent.com/assets/3378822/12281209/03a8ef1a-b9f9-11e5-91bf-ca1670f6d28f.png)

**Comment by johnd0e** [comment]
#1196
